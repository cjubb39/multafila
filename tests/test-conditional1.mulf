/* basic conditional statement, should return 22 */

int main(){
	int x = 5;
	int y = 4;
	int z = 3;
	
	if (x < 10){
		z = x+y+z;
	}
	else {
		z = y+z;
	}
	
	printOut("should return 21");
	int result = x+y+z; /* should return 21 */
}