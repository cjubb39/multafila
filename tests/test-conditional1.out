
/*==========OUTPUT CODE BELOW==========*/
#include <stdio.h>
#include <pthread.h>
void printOut(char *m){printf("%s\n", m);}
void printInt(int m){printf("%d\n", m);}

int main (  );

pthread_mutex_t global_lock;

pthread_cond_t global_cond;

struct global_lock_struct_s{
} global_lock_struct;

pthread_t global_threads [0];
int main (  )
{

{
pthread_mutex_init( & global_lock , NULL);
pthread_cond_init( & global_cond , NULL);
}
 ;
x = 5 ;
y = 4 ;
z = 3 ;
if(x < 10 )
{
z = x + y + z ;
}

else
{
z = y + z ;
}
 ;
printOut( "should return 21" ) ;
result = x + y + z ;
}

